spec/order_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'order'
+
+describe Order do
+
+  context 'Menu' do
+    it 'Should display the default menu' do
+      expect(subject.menu).to eq Order::DEFAULT_MENU
+    end
+  end
+
+  context 'Order' do
+    it 'User can order select dishes' do
+      subject.select('pizza', 3)
+      expect(subject.order).to eq([{ meal: 'pizza', quantity: 3, total: 18 }])
+    end
+
+    it 'Returns current total when order is selected' do
+      expect(subject.select('pizza', 3)).to eq 'Your order will cost £18'
+    end
+
+    it 'returns current total when multiple items are selected' do
+      subject.select('pizza', 3)
+      expect(subject.select('nuggets', 2)).to eq 'Your order will cost £26'
+    end
+  end
+end
```

Comment:
As we've just discussed, there should be a test to test the send_text method that stubs out the response - more details in the code review rubric file